azure-pipelines/ui-automation/templates/run-on-firebase.yml

```diff
@@ -73,14 +53,14 @@ jobs:
               --app "${{ parameters.automationAppApkPath }}" \
               --test "${{ parameters.automationAppTestApkPath }}" \
               --device "model=${{ parameters.firebaseDeviceId }},version=${{ parameters.firebaseDeviceAndroidVersion }}" \
-              --timeout "$(firebaseTimeout)" \
+              --timeout "${{ parameters.firebaseTimeout }}" \
```

Comment:
I do not see firebaseTimeout defined here

Reply:
it is defined in line 16